src/components/CurrencyInput.vue

```diff
@@ -130,13 +130,14 @@ export default {
       }
     }
   },
+  emits: ['update:modelValue', 'input', 'blur'],
```

Comment:
Please break these down with each on its own line.

Reply:
Sure, I just left a comment that I had these changes from before the linting rules, didn't run the linter on them to keep the diff small, I was thinking we could bring this in and then run the linter on all files in one pull request without logic changes to make it easier. But could just lint too in this pr if preferable

Reply:
I added that here though, the last commit is just linting rules, the component commits show the logic changes alone